src/controllers/features/common/typeOfBusinessController.ts

```diff
@@ -15,21 +15,28 @@ import { getAcspRegistration } from "../../../services/acspRegistrationService";
 import { AcspData } from "@companieshouse/api-sdk-node/dist/services/acsp";
 import { ErrorService } from "../../../services/errorService";
 import { AcspDataService } from "../../../services/acspDataService";
+import { getPreviousPageUrl } from "../../../services/url";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
     const lang = selectLang(req.query.lang);
     const locales = getLocalesService();
     const session: Session = req.session as any as Session;
     const previousPage: string = addLangToUrl(BASE_URL, lang);
     const currentUrl: string = BASE_URL + TYPE_OF_BUSINESS;
+
     try {
+
         let typeOfBusiness = "";
         if (session?.getExtraData("resume_application")) {
             // get data from mongo and save to session
             const acspData = await getAcspRegistration(session, session.getExtraData(SUBMISSION_ID)!, res.locals.applicationId);
             if (acspData !== undefined) {
                 saveDataInSession(req, USER_DATA, acspData);
-                if (acspData.typeOfBusiness === "UNINCORPORATED" || acspData.typeOfBusiness === "CORPORATE_BODY") {
+                const prevurl = await getPreviousPageUrl(req, BASE_URL);
+                const defaultBusinessType: boolean = prevurl?.includes(LIMITED_BUSINESS_MUSTBE_AML_REGISTERED_KICKOUT);
```

Comment:
const defaultBusinessType -  constant name is not in sync with what its doing.
 It could be `isPreviousURLLimited.....`